### ISSUE TITLE
NullPointerException in ASTNode

### ISSUE THREAD

**Issue by draeger** [sbmlteam/jsbml#258]
The following issue has been raised as a consequence of failing tests of the SBMLTestSuite using SBSCL (the Systems Biology Simulation Core Library). As it turns out, the problem goes back to a NPE in ASTNode, see https://github.com/draeger-lab/SBSCL/issues/74, and needs to be fixed with a new version of JSBML.

**Comment by draeger** [comment]
Ah, I see: This needs to be fixed in JSBML, and a new release of that library becomes urgent in Maven for this to take effect.

**Comment by dyrpsf** [comment]
> Ah, I see: This needs to be fixed in JSBML, and a new release of that library becomes urgent in Maven for this to take effect.

@draeger,
Yes, exactly – the fix is fully in JSBML (`AbstractTreeNode`/`ASTNode`).  
Once a new JSBML release is published, SBSCL will only need to update its JSBML dependency.
